images/docker-stacks-foundation/Dockerfile

```diff
@@ -73,6 +73,8 @@ RUN sed -i 's/^#force_color_prompt=yes/force_color_prompt=yes/' /etc/skel/.bashr
 RUN echo "auth requisite pam_deny.so" >> /etc/pam.d/su && \
     sed -i.bak -e 's/^%admin/#%admin/' /etc/sudoers && \
     sed -i.bak -e 's/^%sudo/#%sudo/' /etc/sudoers && \
+    # Delete existing user 'ubuntu' with UID=1000
+    userdel -Rz ubuntu && \
```

Comment:
Can you use long options here?

I also think you made a mistake and wanted `-r` (small letter) to remove the home directory and `-Z` (capital letter) to remove SELinux user mapping.